test/fixtures/runs.yml

```diff
@@ -12,7 +12,7 @@ hear_me_response:
   model: gpt-4
   instructions: You are a helpful assistant
   additional_instructions:
-  tools: [{"type": "retrieval"}, {"type": "code_interpreter"}]
+  tools: [{"type": "code_interpreter"}, {"type": "retrieval"}, {"type": "function"}]
```

Comment:
A small bug in the fixtures: these tools should have matched the tools of the assistant. Correcting it.